ch_tools/chadmin/internal/table.py

```diff
@@ -249,6 +260,136 @@ def delete_table(
     )
 
 
+def _is_valid_uuid(uuid_str):
+    try:
+        val = uuid.UUID(uuid_str)
+    except ValueError:
+        return False
+    return str(val) == uuid_str
+
+
+def _get_uuid_table(table_metadata_path: str) -> str:
+    table_uuid = ""
+
+    with open(table_metadata_path, "r", encoding="utf-8") as metadata_file:
+        for line in metadata_file:
+            if "UUID" in line:
+                parts = line.split()
+                try:
+                    uuid_index = parts.index("UUID")
+                    # check size
+                    table_uuid = parts[uuid_index + 1].strip("'")
+                except ValueError:
+                    raise RuntimeError(
+                        f"Failed parse UUID from '{table_metadata_path}'"
+                    )
+                break
+        else:
+            raise RuntimeError(f"No UUID in '{table_metadata_path}'")
+
+    assert _is_valid_uuid(table_uuid)
+
+    return table_uuid
+
+
+def check_table_dettached(ctx, database_name, table_name):
+    query = """
+        SELECT
```

Comment:
I just wonder, is it possible to remove the detached table totally without queries to the clickhouse? For example for the scenario when clickhouse-server can't starts.  Will be useful such use-case?

Reply:
> is it possible to remove the detached table totally without queries to the clickhouse
No.
Firstly we need make sure that table is realy detached so we need send a querry. 

> Will be useful such use-case?
I think no. If clickhouse-server doesn't start, its mean that problem doesn't relating with detached table and need to fix another problem.